os/include/tinyara/debug/sysdbg.h

```diff
@@ -0,0 +1,126 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#ifndef __INCLUDE_DEBUG_SYSDBG_H
+#define __INCLUDE_DEBUG_SYSDBG_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <tinyara/sched.h>
+#include <tinyara/clock.h>
+
```

Comment:
Let's include below:
<sys/types.h> to use pid_t.
<stdint.h> to use uint8_t
<semaphore.h> to use sem_t

Reply:
Ok Done